src/core/shape_inference/include/element_visitor.hpp

```diff
@@ -8,10 +8,17 @@
 
 #include "openvino/core/except.hpp"
 #include "openvino/core/type/element_type.hpp"
+#include "openvino/itt.hpp"
 
 namespace ov {
-namespace element {
+namespace itt {
+namespace domains {
+OV_ITT_DOMAIN(ov_eval);
```

Comment:
is this domain used somewhere?

Reply:
Yes is used to generate `IF_TYPE_OF` macro during CC analysis.